packages/core/addon/components/table-renderer-vertical-collection.js

```diff
@@ -178,6 +185,21 @@ export default Component.extend({
     }
   },
 
+  /**
+   * Event handler for wheel events in header to scroll the table body vertically.
+   * Works for mouse wheels and vertical scrolling via mousepad gestures.
+   * @param {Event} event - wheel event
+   * @private
+   * @returns {Boolean} false
+   */
+  _headerWheelSync(event) {
+    let table = get(this, 'tableWrapperDomElement');
+
+    table.scrollTo(table.scrollLeft + event.deltaX, table.scrollTop);
+    event.preventDefault(); // Prevents the page navigation gesture in Mac OSX
```

Comment:
Do we need both `event.preventDefault()` and `return false`?

Reply:
Nope, just me remembering old fashioned event interface, bad habit. Removed.